tutorial-server/notebooks/CactusTutorial.ipynb

```diff
@@ -101,7 +101,21 @@
     "os.environ[\"OMPI_MCA_rmaps_base_oversubscribe\"] = \"true\"\n",
     "os.environ[\"OMPI_MCA_hwloc_base_binding_policy\"] = \"none\"\n",
     "os.environ[\"OMPI_MCA_btl_vader_single_copy_mechanism\"] = \"none\"\n",
-    "import scrolldown"
+    "#import scrolldown\n",
+    "\n",
+    "# We are going to install kuibit, a Python package to post-process Cactus simulations.\n",
+    "# We will install kuibit inside the Cactus directory. The main reason for this is to\n",
+    "# have a make easier to uninstall kuibit (you can just remove the Cactus folder). \n",
+    "os.environ[\"PYTHONUSERBASE\"] = os.path.join(os.environ[\"HOME\"], \"/Cactus\")\n",
+    "\n",
+    "# We will need to find the path of the directory where the simulation data is saved.\n",
+    "# This is a convenience function to do this in full generality. \n",
+    "# This is here to keep the tutorial working on several different setups. \n",
+    "def get_datadir():\n",
+    "    # Ask simfactory where the latest output was writted\n",
+    "    datadir = os.popen(\"./simfactory/bin/sim get-output-dir tov_ET\").read().rstrip(\"\\\\n\")\n",
+    "    # Remove the /output-XXX part\n",
+    "    return os.path.split(datadir)[0]"
```

Comment:
I updated the code, but I find this function extremely ugly. Does simfactory have any API that we could use for this? What I mean is: would it be possible to import simfactory as a module and get information from there?

Reply:
There is an API but it is not something that is really exposed. The only code that I know of using it is the `distribute` utility in `simfactory/bin` and it has to do a lot of magic to make things work.

There's also no Python function that would return the string that `get-output-dir` outputs. If you are interested, the function that handles that command is `command_get_output_dir` in `lib/sim-manage.py` which is the module to import.

Reply:
I am not a fan of messing with PYTHONUSERBASE. It's a complication I'd rather not see. Just pip install?

Reply:
While I agree that it would be nice to keep things clean in users' environments, I am of the same opinion as @stevenrbrandt .

My take is that the more "infrastructure" functions we add, the harder it becomes to understand how to really use these tools.

Reply:
With multiple different opinions we usually go by majority vote, so please go ahead with pip.